src/utils/cacheRequest.js

```diff
@@ -63,38 +64,34 @@ const fetchCachedGeneExpressionWork = async (experimentId, timeout, body, backen
   return responseData;
 };
 
-const fetchCachedWork = async (experimentId, timeout, body, backendStatus) => {
+const fetchCachedWork = async (
+  experimentId,
+  timeout,
+  body,
+  qcPipelineStartDate,
+  ...additionaSendWorkParams) => {
```

Comment:
This variable is often called `extras` in JavaScript, let's rename it to be consistent.

Reply:
renamed to `extras`